### PR TITLE
fix: no active datanode when frontend start

### PR DESCRIPTION
**Pull request by Fengys123**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Problems encountered:
When starting greptimedb cluster,  frontend will report an error, “no active datanode”. In addition metasrv uses loadbase selector. 

In the current loadbase selector implementation, when the region count of a datanode cannot be obtained, we will exclude the datanode.

When frontend starts, some system tables need to be created. At this time, metasrv does not receive the heartbeat of datanode. So, frontend report an error "no active datanode".

This PR mainly fix this problem.

When the region number of the datanode cannot be obtained and the datanode alive, we give the node largest region count and lower its priority.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by shuiyisong — APPROVED** [review]
LGTM

**Review by fengjiachun — APPROVED** [review]
lgtm

File: src/meta-srv/src/selector/load_based.rs [hunk 1]
```diff
@@ -24,6 +25,8 @@ use crate::lease;
 use crate::metasrv::Context;
 use crate::selector::{Namespace, Selector};
 
+const LARGEST_REGION_NUMBER: u64 = u64::MAX;
```
Comment (killme2008) [inline src/meta-srv/src/selector/load_based.rs:28]:
```suggestion
const MAX_REGION_NUMBER: u64 = u64::MAX;
```

File: src/meta-srv/src/selector/load_based.rs [hunk 2]
```diff
@@ -81,26 +94,26 @@ impl Selector for LoadBasedSelector {
     }
 }
 
-fn to_lease_key(k: &StatKey) -> LeaseKey {
-    LeaseKey {
+fn to_stat_key(k: &LeaseKey) -> StatKey {
```
Comment (killme2008) [inline src/meta-srv/src/selector/load_based.rs:97]: Impl `From` trait for `StatKey` looks better.

**Pull request merged**